### PR TITLE
Add progress bar and keyboard shortcuts

### PR DESCRIPTION
**Pull request by rachita19082**
Fix issue #22 and #23

### CODE REVIEW

**Comment by rachita19082** [comment]
Fix issue #22 in commit 9eb4239.

**Comment by rachita19082** [comment]
Completed task 1 in issue #23 in commit 359e4a6.

Mapped keys as follows:
`Yes` - Up Arrow
`No` - Down Arrow
`Next` - Right Arrow
`Back` - Left Arrow

**Comment by rachita19082** [comment]
Complete task 3 in issue #21 in commit c636a85

**Comment by rachita19082** [comment]
Completed task 1 and 6 in [this comment](https://github.com/unicef/publicgoods-scripts/issues/20#issuecomment-869605459) of #20 in commit 69a4236.

**Comment by rachita19082** [comment]
Completed tasks 2, 3, and 4 in this comment of #20 in commit 3e4af81.

**Comment by rachita19082** [comment]
Completed task 5 in [this comment](https://github.com/unicef/publicgoods-scripts/issues/20#issuecomment-869605459) of #20 in commit ee78ec6. 

Improved responsiveness of buttons and alignment of titles in Summary Page.

**Comment by rachita19082** [comment]
Completed tasks 1-5 in [this comment](https://github.com/unicef/publicgoods-scripts/issues/25#issuecomment-869608920) of #25 in commit 6d59bf3.

Updated form copy in `FAQ section`, removed `X Resources` section wherever copy is not available and update copy in `Summary page`.

**Comment by rachita19082** [comment]
Completed task 2 of #23 in commit e1ce98a.

**Comment by rachita19082** [comment]
@lacabra This pull request is ready for review now. I have fixed issues #22 and #23 and made improvements based on the [feedback](https://github.com/unicef/publicgoods-roadmap/issues/37) by @amreenp7.

Please let me know the changes required in this. Thanks.

File: packages/eligibility/src/Eligibility.js [hunk 1]
```diff
@@ -19,6 +19,20 @@ function Eligibility() {
...
+  React.useEffect(() => {
+    document.addEventListener('keydown', handleKeys);
+
+    return () => {
+      document.removeEventListener('keydown', handleKeys);
+    };
+  }, []);
+
+  function handleKeys(e){ 
+    console.log("Enter handle keys : " + e.keyCode);
```
Comment (lacabra) [inline packages/eligibility/src/Eligibility.js:31]: If this is production-ready code, kindly remove `console.log()` statements.

**Review by lacabra — CHANGES_REQUESTED** [review]
This looks great @rachita19082, thanks for your great work on this! Only requested one very minor change. Thanks!

File: packages/eligibility/src/Eligibility.js (hunk 1 above)
Comment (rachita19082) [reply packages/eligibility/src/Eligibility.js]: Resolved in commit 8ca0f51.

Thank you @lacabra!

**Review by lacabra — APPROVED** [review]

**Pull request merged**